### PR TITLE
Javadoc fixes

### PR DESCRIPTION
**Pull request by kaspersorensen**
This PR fixes a bunch of errors in the javadoc comments of MetaModel. While running `mvn javadoc:javadoc` on a Java 8 runtime, then these bring build failures.

### CODE REVIEW

File: jdbc/src/main/java/org/apache/metamodel/jdbc/dialects/DB2QueryRewriter.java [hunk 1]
```diff
@@ -69,11 +69,11 @@ public String rewriteQuery(Query query) {
         final Integer firstRow = query.getFirstRow();
         final Integer maxRows = query.getMaxRows();
 
-        if (maxRows == null && firstRow == null) {
+        if (maxRows == null && (firstRow == null || firstRow.intValue() == 1)) {
```
Comment (albertostratio) [inline jdbc/src/main/java/org/apache/metamodel/jdbc/dialects/DB2QueryRewriter.java]: Shouldn't this be included in another PR?

Comment (kaspersorensen) [inline jdbc/src/main/java/org/apache/metamodel/jdbc/dialects/DB2QueryRewriter.java]: Yes, but something went wrong - somehow @ankit2711's commit went into my PR ... Don't know why... at least it's not "my" change and wasn't my intention ... sorry, will repost I guess.

**Comment by kaspersorensen** [comment]
Hmm seems something is odd with the github mirror ... The commit has been made on our primary git repo, but not synced to master on github. See for yourself: https://git-wip-us.apache.org/repos/asf?p=metamodel.git

**Comment by kaspersorensen** [comment]
Now it should be updated, thanks to INFRA. Can you give it an updated look @albertostratio ?

**Comment by albertostratio** [comment]
LGTM @kaspersorensen
